### PR TITLE
Update also docs/antora.yml when preparing a release

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: .github/workflows/prepare-release.yml [hunk 1]
```diff
@@ -92,11 +92,16 @@ jobs:
           MAVEN_EXEC: ${{ inputs.maven_executable }}
 
       - name: Update latest release version in docs
-        if: ${{ hashFiles('docs/pom.xml') != '' }}
+        if: ${{ hashFiles('docs/pom.xml') != '' || hashFiles('docs/antora.yml') }}
```
Comment (gastaldi) [inline .github/workflows/prepare-release.yml]: There are some projects without a pom.xml in the `docs/` module, maybe use `&&`?

Comment (gastaldi) [reply .github/workflows/prepare-release.yml]: Or perhaps `git add docs` directly

Comment (ppalaga) [reply .github/workflows/prepare-release.yml]: Would you like to adapt it? What I'd need is to add the changed antora.yml. I am far from knowing what different setups other projects use.

Comment (gastaldi) [reply .github/workflows/prepare-release.yml]: Sure, I'll do it first thing Monday morning 👍🏻 
Thanks!

**Comment by gastaldi** [comment]
@ppalaga I went ahead and fixed that earlier than expected. Enjoy 😉

**Comment by ppalaga** [comment]
Thanks a lot, @gastaldi!

**Pull request merged**